internal/metrics/synthetics/agent.go

```diff
@@ -139,7 +139,11 @@ func isFlyInfraTarget(target string) bool {
 		host = parsedURL.Host
 	}
 
-	_, existsInFlyInfraDomains := flyInfraHosts[host]
+	for _, flyInfraDomain := range flyInfraDomains {
+		if strings.HasSuffix(host, "."+flyInfraDomain) || host == flyInfraDomain {
```

Comment:
Could just return the evaluation here I think?

Reply:
Hmmm, not sure I fully get the comment 😅 I _think_ I am . If the target host ends with any of the allowed domains or is the naked domain itself I'm returning the evaluation as positive, if the loop ends and it haven't returned for any of the allowed domains I return a nope ... What am I missing 🤔

Reply:
oh. nevermind i was looking at this on my phone. NEVERMIND ME.